### PR TITLE
ScheduledReminders - Refactor form to work at a standalone url

### PR DESCRIPTION
**Pull request by colemanw**
Overview
----------------------------------------
One more form in an ongoing effort to decouple pages from forms. This one is for Scheduled Reminders.

Before
----------------------------------------
Form coupled with page. Exhibits bug in https://github.com/civicrm/civicrm-core/pull/26344

After
----------------------------------------
Form and page decoupled following standard pattern used for other admin forms. Bug fixed as a byproduct (https://lab.civicrm.org/dev/core/-/issues/4309).

### CODE REVIEW

File: CRM/Core/DAO/ActionSchedule.php [hunk 1]
```diff
@@ -37,6 +37,17 @@ class CRM_Core_DAO_ActionSchedule extends CRM_Core_DAO {
    */
   public static $_log = FALSE;
 
+  /**
+   * Paths for accessing this entity in the UI.
+   *
+   * @var string[]
+   */
+  protected static $_paths = [
+    'add' => 'civicrm/admin/scheduleReminders/edit?reset=1&action=add',
```
Comment (totten) [inline CRM/Core/DAO/ActionSchedule.php:46]: If you look at the main listing page (`/civicrm/admin/scheduleReminders?reset=1`), the `Add Reminder" button still goes to the old URL (e.g. `http://core-26884-62nfn.test-1.civicrm.org:8001/civicrm/admin/scheduleReminders?action=add&reset=1`). Shouldn't that change to match?

**Comment by totten** [comment]
On the "Edit" screen on the autobuild test site, I see this warning - which I don't remember seeing before:

```
User warning: Smarty error: unable to read resource: "CRM/Admin/Form/ScheduleReminders.hlp" in Smarty->trigger_error() (line 1100 of /home/jenkins/bknix-dfl/build/core-26884-62nfn/web/sites/all/modules/civicrm/packages/Smarty/Smarty.class.php).
```

**Comment by totten** [comment]
OK, cool, that is better. 

I tested a bit with multi-tab usage (as in dev/core#4309). Each tab stores the data as you would expect. Woot.

One last thing, though -- after you submit the form, where should you go? Before, it returned to the listing screen (`civicrm/admin/scheduleReminders`) -- now, it goes to the site-root (`/`). Granted, most people probably don't open this page in secondary tabs. But if the site is set to `ajaxPopupsEnabled=0`, then the redirect feels more problematic.

**Comment by colemanw** [comment]
Thanks for the review @totten - I'm working on a metadata-driven solution for the redirects that can apply to every CRM_Admin_Form, not just this one, because I see all the decoupled forms now have this problem.

**Pull request merged**